onboarder/installation/index.rst

```diff
@@ -65,6 +70,19 @@ Make sure this file is shipped to the server which is hosting your PIM.
 
 This file must be accessible (read rights) by the system user that runs the pim (example: www-data).
 
+Check your .env file
+--------------------
+
+.. code-block:: bash
+
+    APP_ENV=prod
```

Comment:
Same question here, @momoss Do you know why we are asking the customer to define those variables? If they are not defined, the PIM does not work at all. It is not related to the Onboarder.

Reply:
It's related to the MessageRegistry launched in sub process by MessageConsumer using proc_open, the ENV variables are not well populated in the subprocess

Reply:
This behavior appear since a PR of Damien removing the use of Symfony Process component in favour of proc_open to better handle STDERR and STDOUT

Reply:
Sorry, I still don't understand why we are asking the customer to redefine those variables in the Onboarder doc, they are already define while installing the PIM